### PR TITLE
Solaris publishing fixes

### PR DESCRIPTION
**Pull request by sersut**
This PR fixes https://github.com/chef/omnitruck/issues/176 and https://github.com/chef/omnitruck/issues/175. 

For https://github.com/chef/omnitruck/issues/176 we need to return `chef.bintray.com` based URLs due to the openssl version in that solaris version. 

For https://github.com/chef/omnitruck/issues/175 looks like for 4 builds we have different platform and platform_version values fir published solaris artifacts `solaris` instead of `solaris2` and `10` instead of `5.10`. You can check these out at http://chef.bintray.com/stable/solaris/ and http://chef.bintray.com/stable/solaris2/. 

/cc: @chef/engineering-services

### CODE REVIEW

File: lib/mixlib/install/backend/bintray.rb [hunk 1]
```diff
@@ -259,6 +260,29 @@ def parse_platform_info(artifact_map)
         end
 
         #
+        # Normalizes platform and platform_version information that we receieve
```
Comment (scotthain) [inline lib/mixlib/install/backend/bintray.rb]: `receive`

**Comment by scotthain** [comment]
Super minor typo - otherwise 👍

**Comment by patrick-wright** [comment]
that's why this logic exists :) There's a change we'll be adding more
👍

**Comment by sersut** [comment]
Fixed the typo 😄 Thanks @scotthain

**Comment by yzl** [comment]
👍

**Pull request merged**